students/cindywaldron/lesson9/mailroom.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+class Donor:
+
+    def __init__(self, name, amount):
+        self._name = name
+        self._amount = amount
+        self._donation_count = 1
+
+    @property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, name):
+        self._name = name
```

Comment:
property/setters are great when you would like to apply extra logic to attributes, otherwise regular class attributes would be better here (less unnecessary boilerplate)